components/org.wso2.carbon.identity.idp.metadata.saml2/src/main/java/org/wso2/carbon/identity/idp/metadata/saml2/util/SAMLMetadataConverter.java

```diff
@@ -83,10 +83,10 @@ public boolean canHandle(Property property) {
     /**
      * Retrieves whether the SAML IDP is available in registry or not.
      *
-     * @param tenantId ID of the tenant domain
-     * @param idPName  ID of the IDP which is needed to be checked whether it has a metadata string in registry
-     * @return a boolean regarding to the condition
-     * @throws IdentityProviderManagementException if there is an error in the registry access
+     * @param tenantId ID of the tenant domain.
+     * @param idPName  ID of the IDP which is needed to be checked whether it has a metadata string in registry.
+     * @return a boolean regarding to the condition.
```

Comment:
Shall we start from an uppercase letter?

Shall we start as "Returns true if ....". WDYT?